api/apps/geoprocessing/src/modules/scenarios/scenarios.service.ts

```diff
@@ -0,0 +1,78 @@
+import { Injectable, Logger, Inject } from '@nestjs/common';
+import {
+  TileService,
+  TileRequest,
+} from '@marxan-geoprocessing/modules/tile/tile.service';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import {
+  IsString,
+} from 'class-validator';
+
+import { ScenariosPuPaDataGeo } from '@marxan/scenarios-planning-unit/scenarios-pu-pa-data.geo.entity';
+
+export class ScenariosPUFilters {
+}
+
+export class ScenariosTileRequest extends TileRequest {
+  @IsString()
+  id!: string;
+}
+
+@Injectable()
+export class ScenariosService {
+  private readonly logger: Logger = new Logger(ScenariosService.name);
+  constructor(
+    @InjectRepository(ScenariosPuPaDataGeo)
+    private readonly ScenariosPlanningUnitGeoEntityRepository: Repository<ScenariosPuPaDataGeo>,
+    @Inject(TileService)
+    private readonly tileService: TileService,
+  ) {}
+
+  /**
+   * @param filters bounding box of the area where the grids would be generated
```

Comment:
maybe add a `@todo` note if this needs to be implemented later on (or outright delete the method if we won't need it)

Reply:
i did deleted as under scenarios we should no longer filter by bbox as the data should be only refer to the Planning area location so, i will delete it for good